_util/build_mobile.mk

```diff
@@ -60,7 +60,7 @@ build-iossimulator:
 
 build-ios: 
 	@echo "Building iOS framework. Please wait..."
-	@CGO_CFLAGS=$(MINIOSVERSIONMIN) gomobile bind -v -ldflags="-s -w" -target=ios/arm64,iossimulator/amd64 -tags "ios mobile" -o $(IOSMOBILESDKDIR)/ios/$(IOSBINNAME) $(PKG_EXPORTS)
+	@CGO_CFLAGS=$(MINIOSVERSIONMIN) gomobile bind -v -ldflags="-s -w" -target=ios,iossimulator -tags "ios mobile" -o $(IOSMOBILESDKDIR)/ios/$(IOSBINNAME) $(PKG_EXPORTS)
```

Comment:
@aaryankotharii  I remember it doesn't work. because ios and iossimulator both has arm64. that is why we split them into two build tasks.

Reply:
@cnlangzi I'm not sure why that issue was. but I tested with the build on this pr on both m1 Mac and intel using phone and simulator both. working on all cases.